### PR TITLE
Remove outdated docs from lisk sdk - Closes 1655 

### LINKED ISSUE #1655: Remove outdated docs from lisk-sdk
### Description
Not updated content from SDKv5 needs to be removed.

#### Affected branch / URL
`docs-sdk`

#### Page location
- events
- actions
- modules
- plugins
- Complete references category

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1655

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -46,13 +48,17 @@ The most important highlights and features of the Lisk SDK:
...
+*** Bootstrap a complete, default blockchain application with a single command.
 *** Generate skeletons for modules, assets, and plugins.
 *** Use the Lisk SDK libraries directly in the terminal via the interactive Lisk console.
 *** Many other convenient CLI tools such as creating accounts, encrypting passphrases/messages, and more.
-** xref:{url_references_dashboard_plugin}[]: Communicate conveniently with your blockchain application during development via a dashboard.
-** xref:{url_references_test_suite}[SDK testing utilities]: simplifies the testing of all components of a blockchain application by providing dedicated fixtures, mocks, and utility functions.
+// ** xref:{url_references_dashboard_plugin}[]:
+Dashboard Plugin: Communicate conveniently with your blockchain application during development via a dashboard.
+// ** xref:{url_references_test_suite}[SDK testing utilities]:
+** SDK testing utilities: simplifies the testing of all components of a blockchain application by providing dedicated fixtures, mocks, and utility functions.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
** SDK testing utilities: simplifies the testing of all components of a blockchain application by providing dedicated fixtures, mock-ups, and utility functions.
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -73,12 +79,15 @@ The Lisk SDK operates on the {nodejs}[Node.js^]. Its three components Lisk Frame
 [width="100%",cols="23%,77%",options="header",]
 |===
 | Component | Description
-| xref:{url_references_framework}[Lisk Framework] | Lisk Framework is an application framework responsible for establishing and maintaining the interactions between the different components of a blockchain application.
+| Lisk Framework | Lisk Framework is an application framework responsible for establishing and maintaining the interactions between the different components of a blockchain application.
+// xref:{url_references_framework}[Lisk Framework] 
 
-| xref:{url_references_elements}[Lisk Elements] | Lisk Elements is a collection of libraries, each of which implements a certain type of blockchain application functionality such as cryptography, transactions, p2p, etc.
+| Lisk Elements | Lisk Elements is a collection of libraries, each of which implements a certain type of blockchain application functionality such as cryptography, transactions, p2p, etc.
 Each library of Lisk Elements is also published as NPM package, and can be installed separately, if needed.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Each library of Lisk Elements is also published as an NPM package, and can be installed separately if needed.
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -73,12 +79,15 @@ The Lisk SDK operates on the {nodejs}[Node.js^]. Its three components Lisk Frame
...
 For more information about Lisk Protocol, see {url_protocol_lips}[Lisk Improvement Proposals].
+// xref:{url_references_elements}[Lisk Elements]
 
-| xref:{url_references_commander}[Lisk Commander] | Lisk Commander is a command line tool that provides various commands to simplify the development and management of blockchain applications.
+| Lisk Commander | Lisk Commander is a command line tool that provides various commands to simplify the development and management of blockchain applications.
 For example, it allows to xref:{url_guides_setup}[bootstrap] a complete blockchain application with just one command.
```
Comment (chris529) [inline modules/ROOT/pages/index.adoc]:
```suggestion
For example, it allows the user to xref:{url_guides_setup}[bootstrap] a complete blockchain application with just one command.
```

**Review by chris529 — COMMENTED** [review]
Looks good! Just a few very minor points 👍

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**